zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/bpmn/activity/CallActivityInterruptionTest.java

```diff
@@ -61,6 +62,10 @@ public void init() {
   }
 
   @Test
+  @Ignore(
+      "Temporarily disabled due to changes in "
+          + "`io.camunda.zeebe.engine.processing.bpmn.event.EndEventProcessor.NoneEndEventBehavior` "
+          + "that provided support for Execution Listeners")
   public void shouldTriggerBoundaryEventIfChildIsCompleting() {
```

Comment:
Hey @saig0, 
I wanted to bring your attention to this test. 
After making changes to the `EndEventProcessor.NoneEndEventBehavior` to support ExecutionListeners, we now write an additional command (`END_EVENT, COMPLETE_ELEMENT`) and it has caused this test to fail.

Here is the diff between the call activity subprocess record logs (https://www.diffchecker.com/a6JKYMby/):
- **Before changes: (left)**
  - ("wf-child-end", END_EVENT, ELEMENT_ACTIVATED),
  - ("wf-child-end", END_EVENT, ELEMENT_COMPLETING),
  - ("wf-child-end", END_EVENT, ELEMENT_COMPLETED),
  - ("wf-child", PROCESS, COMPLETE_ELEMENT)
  - ("wf-child", PROCESS, TERMINATE_ELEMENT)
  - ("wf-child", PROCESS, ELEMENT_COMPLETING)
  - ("wf-child", PROCESS, ELEMENT_COMPLETED)
  - ("wf-child", PROCESS, TERMINATE_ELEMENT)

- **After changes: (right)**
  - ("wf-child-end", END_EVENT, ELEMENT_ACTIVATED),
  - ("wf-child-end", END_EVENT, COMPLETE_ELEMENT),
  - ("wf-child", PROCESS, TERMINATE_ELEMENT)
  - ("wf-child-end", END_EVENT, ELEMENT_COMPLETING),
  - ("wf-child-end", END_EVENT, ELEMENT_COMPLETED),
  - ("wf-child", PROCESS, COMPLETE_ELEMENT)
  - ("wf-child", PROCESS, ELEMENT_TERMINATING)
  - ("wf-child", PROCESS, ELEMENT_TERMINATED)
  - ("wf-child", PROCESS, COMPLETE_ELEMENT)

Could you please help me resolve this situation, it looks a bit confusing to me?

Reply:
:wrench: We could do the following:

- Remove the end event from the child process
- Change the order of the commands: first trigger the timer, then complete the child element
- Be happy. The record sequence matches again. :tada:

Reply:
Cool, the proposal was applied!
Thanks!)